api/src/main/java/org/openmrs/OrderGroup.java

```diff
@@ -228,4 +230,24 @@ public OrderGroup getParentOrderGroup() {
 	public void setParentOrderGroup(OrderGroup parentOrderGroup) {
 		this.parentOrderGroup = parentOrderGroup;
 	}
+
+	/**
+	 * Gets the orderGroupReason which denotes the reason why the Group was Ordered
```

Comment:
Is `orderGroupReason` supposed to be a variable name? Or a proper English sentence?

Reply:
its a proper english name

Reply:
Javadocs are supposed to be proper english grammar.

Reply:
thanks,, i have corrected it